docs/examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Examples
```

Comment:
I can't reach this page from mkdocs, and this folder currently in .gitinore, because examples are auto-generated. So, I'm not sure how to proceed with it.